metadata-ingestion/tests/unit/cli/dataset/test_resources/dataset.yaml

```diff
@@ -22,6 +22,9 @@
   platform: hive
   urn: urn:li:dataset:(urn:li:dataPlatform:hive,user.clicksv3,PROD) # use urn instead of id and platform
   subtype: View
+  name: clicks v3
```

Comment:
if we can add this and have no other test changes, then our tests are really not doing anything. we probably should be comparing the output to a golden mcp file.

If you need to unblock in the short term, you can use the wheel builds